packages/ui-extensions/src/surfaces/admin/components/CustomerSegmentTemplate/CustomerSegmentTemplate.ts

```diff
@@ -0,0 +1,33 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+/**
+ * Reserved namespace and key for the customer standard metafield used in the template's query.
+ * More info - https://shopify.dev/docs/apps/custom-data/metafields/definitions/standard
+ */
+type CustomerStandardMetafieldDependency = 'facts.birth_date';
+
+export interface CustomerSegmentTemplateProps {
+  /* Localized title of the template. */
+  title: string;
+  /* Localized description(s) of the template. */
+  description: string | string[];
+  /* ShopifyQL code snippet to render in the template with syntax highlighting. */
```

Comment:
Nitpicky but it is it actually ShopifyQL? Notebook uses ShopifyQL we... have a different syntax lol 😅 just wondering if it can cause confusion

Reply:
good catch, I just removed ShopifyQL from them both (so it's just "code snippet...")